src/wp-includes/block-template-utils.php

```diff
@@ -360,6 +360,17 @@ function _get_block_templates_files( $template_type, $query = array() ) {
 				continue;
 			}
 
+			// The child theme items (stylesheet) are processed before the parent theme's (template).
+			// If a child theme defines a template, prevent the parent template from being added to the list as well.
```

Comment:
```suggestion
			/*
			 * The child theme items (stylesheet) are processed before the parent theme's (template).
			 * If a child theme defines a template, prevent the parent template from being added to the list as well.
			 */
```

Use multi line comment.